tests/robot-tests/tests/libs/admin_api.py

```diff
@@ -199,15 +218,48 @@ def user_create_test_release_via_api(
     ), f"Creating release API request failed with {response.status_code} and {response.text}"
 
 
-def get_user_details_via_api(user_email: str):
+def __get_user_details_via_api(user_email: str):
```

Comment:
Only need one underscore to denote this function as private

Reply:
Nice spot - I got my conventions wrong!  Updated.